### ISSUE TITLE
Question: De-prioritization of traffic

### ISSUE THREAD

**Issue by nachefj** [ParsePlatform/parse-server#921]
Hi,

Regarding the following statement in the migration doc: "If you haven't migrated your database by April 28, we will assume your app is low priority, and we will de-prioritize its traffic to focus on production apps."

How will de-prioritization affect Push Notifications? My assumption is that it only affect delivery time but no messages will be dropped.

Also, can there be a request made for an extension?

Thanks,
Joe

**Comment by hramos** [comment]
Hi there, I'm part of the team who is actively helping people migrate their apps. The answer is that we don't know how this may affect push notifications.

When we announced that the hosted Parse service will be shut down next January, we also provided a [migration guide][1] where we have shared a recommended set of guidelines for your migration:

* Migrate your database by April 28th, 2016
* Migrate your API traffic by July 28th, 2016

We are committed to maintaining the Parse API up and running until January of next year, but we highly recommend starting your migration work as soon as possible. We are focusing our efforts on people who are following our recommended instructions.

[1]: https://github.com/ParsePlatform/parse-server/wiki/Migrating-an-Existing-Parse-App
